iFixit/react-commerce
# Use new algolia index

closes #144 

## Changelog

- Use `product_en` Algolia index
- Refactor components to account for more / different data (e.g. review count)
- Update seed data
- Refactor filters tests

## QA

1. Visit the Vercel PR preview
2. Visit parts product list page
3. Verify that filter/search results come from `product_en` Algolia index
